modules/users/tests/server/user.server.model.tests.js

```diff
@@ -77,6 +86,37 @@ describe('User Model Unit Tests:', function() {
 				done();
 			});
 		});
+
+		it('should be able to save 2 different users', function(done) {
+			user.remove(function(err) {
+				should.not.exist(err);
+				user.save(function(err) {
+					user3.save(function(err) {
+						should.not.exist(err);
+						user3.remove(function(err) {
+							should.not.exist(err);
+							done();
+						});
+						
+					});	
+				});
+			});
+		});
+
+		it('should not be able to save different user with the same email address', function(done) {
+			user.remove(function(err) {
+				should.not.exist(err);
+				user.save(function(err) {
+					user3.email = 'test@test.com';
```

Comment:
Should this line be using the email property of user? If for some reason the value of user.email changes, it would be easier to manage by referencing whatever email is being used by the user object.

`user3.email = user.email;`